mdadm.8.in

```diff
@@ -1878,11 +1878,11 @@ script.
 .SH For Monitor mode:
 .TP
 .BR \-m ", " \-\-mail
-Give a mail address to send alerts to.
+Give a mail address to send alerts to. Can be configured in mdadm.conf as MAILADDR.
```

Comment:
shouldn't you care to always **bold** when mentioning `mdadm.conf` at least? It feels like a pattern up until now...

That would apply for most of the changes into the `man`

Reply:
Thanks for review!
Indeed, I missed that because cmdline version of man does not bold it, but I can see that bold is respected correctly via html, for example here:
https://linux.die.net/man/8/mdadm

Reply:
updated